RELEASE_NOTES.md

```diff
@@ -1,6 +1,19 @@
 Release Notes
 =============
 
+v1.0.0-beta.1.x
```

Comment:
[fix] As were adding breaking changes (probably good habit anyway so we think properly at release time).
```suggestion
v1.0.0-beta.x.x
```

Reply:
Updated in 1989a8dc